src/raftstore/store/snap.rs

```diff
@@ -307,9 +313,13 @@ pub struct Snap {
     size_track: Arc<AtomicU64>,
     limiter: Option<Arc<RocksIOLimiter>>,
     hold_tmp_files: bool,
+    _s: PhantomData<S>,
```

Comment:
Why do we need `S` if it is only used in a phantom field?

Reply:
Huh. It is indeed unused. I assumed it was needed because S was used in a method somewhere, but it's not used at all. Weird. I've removed it.

Reply:
Ah, I think that type parameter is required in future patches.